__tests__/app.test.js

```diff
@@ -305,6 +305,27 @@ describe("PATCH /api/articles/:article_id", () => {
         expect(patchedArticle).toEqual(article_1);
       });
   });
+  test("should return 201 code and update an article with minus inputs", () => {
+    const patchInfo = { inc_votes: -50 };
+    const article_1 = {
+      article_id: 1,
+      title: "Living in the shadow of a great man",
+      topic: "mitch",
+      author: "butter_bridge",
+      body: "I find this existence challenging",
+      created_at: "2020-07-09T20:11:00.000Z",
+      votes: 50,
+      article_img_url:
+        "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+    };
+    return request(app)
+      .patch("/api/articles/1")
+      .send(patchInfo)
+      .expect(201)
+      .then(({ body: { patchedArticle } }) => {
+        expect(patchedArticle).toEqual(article_1);
```

Comment:
We don't need to check the whole shape of an object here - just that the vote property has been updated.